setup.cfg

```diff
@@ -35,8 +35,6 @@ classifiers =
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
```

Comment:
there's an additional `Python :: 3 :: Only` classifier that can be useful [setup-cfg-fmt](https://github.com/asottile/setup-cfg-fmt) can add it automatically based on `python_requires`